core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -21,6 +21,46 @@
     html, body {
       background-color: #eee;
     }
+
+    /* TODO(mgowano): Implement horizontal scrolling using Javascript. */
+    .collection-editor-node-list-container {
+      background-color: white;
+      display: block;
+      margin: 30px auto;
+      min-width: 1200px;
```

Comment:
What happens if the viewport is smaller than this? It doesn't seem like the container fits within anything smaller than 1200, which might be an issue for smaller displays.